### PR TITLE
Added logic for notifying PTE webapp for Nightly and PR builds

### PR DESCRIPTION
**Pull request by mstfbl**
This PR adds the logic to notify the PTE webapp for DevOps PyTorch Nightly and PR builds.

### CODE REVIEW

File: .azure_pipelines/job_templates/notify-webapp-template.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  timeoutInMinutes: 600
+  strategy:
+    matrix:
+      ${{ insert }}: ${{parameters.customMatrixes}}
+  pool:
+    name: ${{ parameters.pool}}
+  steps:
+  # Clone PyTorch Tests repository
+  - bash: |
+      B64_PAT=$(printf "%s"":$_ADOTOKEN" | base64)
```
Comment (malfet) [inline .azure_pipelines/job_templates/notify-webapp-template.yml:17]: Can you please explain what are you trying to do here? Do you want to add `:` in front of the token and print it without trailing newline character?
```suggestion
      B64_PAT=$(echo -n ":$_ADOTOKEN" | base64)
```

**Review by malfet — APPROVED** [review]

File: .azure_pipelines/job_templates/notify-webapp-template.yml (hunk 1 above)
Comment (mstfbl) [reply .azure_pipelines/job_templates/notify-webapp-template.yml:17]: Here I am converting $_ADOTOKEN to Base-64 format. Thank you for your suggestion, it is cleaner and I will update this PR as such.

**Comment by Psoffie009** [comment]
> This PR adds the logic to notify the PTE webapp for DevOps PyTorch Nightly and PR builds.

**Comment by Psoffie009** [comment]
> @malfet has imported this pull request. If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30046165).

**Pull request closed without merging**